scripts/test_sluice_metadata.py

```
@@ -0,0 +1,462 @@
+
+
+def test_key_without_use_is_a_service_and_is_enforced(
+    monkeypatch: pytest.MonkeyPatch,
+) -> None:
+    """Every service key in keys.yaml deliberately omits `use:`. Absence must
+    mean enforced, not exempt — the opposite default would silently exempt
+    every new service key."""
+    guard = _guard(monkeypatch, SLUICE_REQUIRE_REQUEST_METADATA="true")
+    key = _FakeKey(alias="dashboard-playground", project="sluice")
+
+    with pytest.raises(Exception):
```

Ruff (0.16.1)

[warning] Do not assert blind exception: `Exception`

(B017)